src/components/VaultRow/lib/VaultDepositForm.tsx

```diff
@@ -196,43 +173,23 @@ export const VaultDepositForm: FC<VaultDepositWithdrawProps> = ({
     overrides: { value: inputIsEth ? value : BigNumber.from(0) },
   })
   const depositUnderlying = useContractWrite(prepareDepositUnderlying.config)
-  const waitDepositUnderlying = useWaitForTransaction({
+  useWaitForTransaction({
     hash: depositUnderlying.data?.hash,
-    onSettled: (receipt, error) =>
```

Comment:
why did you just remove it? No need to show the toast here? You would say you do it in confirmationDetails, but what happens if you're in an expert mode - no toast?

Reply:
The toast component handles transaction settlement